animalfacts.py

```diff
@@ -232,6 +232,7 @@ def check_comment_for_animal(comment, reddit):
     botengine('buffalo', '\sbuffalos?\s', reddit, BUFFALO_FACTS, comment)
     botengine('bobcat','\sbobcats?\s',reddit, BOBCAT_FACTS,comment)
     botengine('bison', '\sbisons?\s', reddit, BISON_FACTS, comment)
+    botengine('blue sea slug', '\s(blue (sea slug|angel|glaucus|dragon|ocean slug))|(sea swallow)?\s', reddit, BLUE_SEA_SLUG_FACTS, comment)
```

Comment:
@joel-g I can't do regex :( is this all good? We have the two word issue #168 to think about first.